docs/user/dql/functions.rst

```diff
@@ -2979,6 +2979,67 @@ Another example to show how to set custom values for the optional parameters::
     +------+--------------------------+----------------------+
 
 
+QUERY
+------------
+
+Description
+>>>>>>>>>>>
+
+``query("[query_expression]+" ,[, option=<option_value>]*)``
+
+The query function maps to the query_string query used in search engine, to return the documents that match a provided text, number, date or boolean value with a given field or fields.
```

Comment:
I would explicitly indicate here that "This is an alternative syntax to the query_string function. "

Reply:
Fixed in https://github.com/Bit-Quill/opensearch-project-sql/pull/143/commits/451ec4ce3ac2f54176734dc61fbc76a3e3cc0c6a